webapp/src/components/backstage/automation/invite_users_selector.tsx

```diff
@@ -38,16 +53,23 @@ const InviteUsersSelector: FC<Props> = (props: Props) => {
 
             //@ts-ignore
             profiles.then(({data}: { data: UserProfile[] }) => {
-                const invitedProfiles: UserProfile[] = [];
-                const nonInvitedProfiles: UserProfile[] = [];
-
-                data.forEach((profile: UserProfile) => {
-                    if (props.userIds.includes(profile.id)) {
-                        invitedProfiles.push(profile);
-                    } else {
-                        nonInvitedProfiles.push(profile);
-                    }
-                });
+                let invitedProfiles: UserProfile[] = [];
+                let nonInvitedProfiles: UserProfile[] = [];
+
+                if (term.trim().length === 0) {
+                    invitedProfiles = invitedUsers.filter((user) => user);
```

Comment:
This appears to just copy `invitedUsers`? Should we make that more explicit with:
```suggestion
                    invitedProfiles = [...invitedUsers];
```

Reply:
I'm filtering out the undefined ones! We can make it more explicit with `user !== undefined`, if you want to.

Reply:
Typescript complains about using undefined. I see even more confusing to add a typescript ignore comment, so I let it as it was. The other cool option is doing `invitedUsers.filter(Boolean)` :sunglasses: But I see that it's equally confusing.

Reply:
The `(u) => u` is fine by me (we do a similar filtering elsewhere in the code). Maybe leave a comment for future us who won't remember this idiom? :)

Reply:
Added a comment in https://github.com/mattermost/mattermost-plugin-incident-collaboration/pull/486/commits/5d0531150a2484814bde10fc027b0f61a0db7e72, resolving this one!